estate/models/users.py

```diff
@@ -0,0 +1,7 @@
+from odoo import fields, models
+
+
+class Users(models.Model):
+    _inherit = "res.users"
+
+    property_ids = fields.One2many("property", "salesperson_id", "Properties")
```

Comment:
I think string is missing?

```suggestion
    property_ids = fields.One2many("property", "salesperson_id", string="Properties")
```